WordPress/Classes/Extensions/UIImage+Assets.swift

```diff
@@ -3,11 +3,10 @@ import Foundation
 
 // MARK: - WordPress Named Assets
 //
-extension UIImage {
-
+@objc
```

Comment:
I'm not sure this one's needed either.

Reply:
[Same](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Extensions/UIImage%2BAssets.swift). :)